site/en/docs/privacy-sandbox/topics/index.md

```diff
@@ -347,629 +89,188 @@ const creative = await response.json();
 
-The **Observed-by context domains (hashed)** column provides the hashed value of a hostname for
-which a topic was observed.
+The **Observed-by context domains (hashed)** column provides the hashed value of a hostname for which a topic was observed.
 
-#### View topics inferred for hostnames {: #view-inferred-topics}
+### View topics inferred for hostnames
 
-You can view the topics inferred by the Topics
-[classifier model](https://github.com/patcg-individual-drafts/topics#:~:text=classifier%20model) for
-one or more hostnames.
+You can also view the topics inferred by the Topics [classifier model](https://github.com/patcg-individual-drafts/topics#:~:text=classifier%20model) for one or more hostnames in `chrome://topics-internals`.
 
+<figure>
 {% Img src="image/80mq7dk16vVEg8BBhsVe42n6zn82/SOTuE2ljC55PaYll1UP1.png",
-  alt="chrome://topics-internal page with Classifier panel selected.",
+  alt="The chrome://topics-internals page with Classifier panel selected.",
   width="800", height="695" %}
+  <figcaption>The chrome://topics-internals page Classifier panel shows topics selected, hosts visited, and model version and path.</figcaption>
+</figure>
 
-{% Aside %}
-The current implementation of the Topics API infers topics from hostnames only: not any other part
-of a URL.
+The current implementation of the Topics API infers topics from hostnames only; not from any other part of a URL.
 
-Use hostnames only (without protocol or path) to view inferred topics from the
-`chrome://topics-internals` Classifier. `chrome://topics-internals` will display an error if you
-attempt to include a  "/" in the Host field.
-{% endAside %}
+Use hostnames only (without protocol or path) to view inferred topics from the `chrome://topics-internals` Classifier. `chrome://topics-internals` will display an error if you attempt to include a "/" in the Host field.
 
-####  View Topics API information {: #view-api-information}
+### View Topics API information
 
-Information is provided about the Topics API implementation and settings, such as the
-[taxonomy](/docs/privacy-sandbox/topics/#taxonomy) version and
-[epoch](/docs/privacy-sandbox/topics/#epoch) duration. These values
-reflect default settings for the API or parameters successfully set [from the command
-line](#feature-flags). This is handy for checking that command line flags have worked as expected:
-in the example below, `time_period_per_epoch` has been set to 15 seconds (the default is seven
-days).
+Information is provided in `chrome://topics-internals` about the Topics API implementation and settings, such as the [taxonomy](https://github.com/jkarlin/topics/blob/main/taxonomy_v1.md) version and epoch duration. These values reflect default settings for the API or parameters successfully set [from the command line](#feature-flags). This is handy for checking that command-line flags have worked as expected. In the example below, `time_period_per_epoch` has been set to 15 seconds (the default is seven days).
```

Comment:
```suggestion
You can find information about the Topics API implementation and settings, such as the [taxonomy](https://github.com/jkarlin/topics/blob/main/taxonomy_v1.md) version and epoch duration, in `chrome://topics-internals`. These values reflect default settings for the API or parameters successfully set [from the command line](#feature-flags). This may be helpful to confirm that command line flags have worked as expected.

  In the example below, `time_period_per_epoch` has been set to 15 seconds (the default is seven days).
```